deploy/backplane/acs/01-acs-admins-project.ClusterRole.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-acs-admins-project
+rules:
+# ACS team can interact with ACS
+- apiGroups:
+  - platform.stackrox.io
+  - cloud.stackrox.io
+  resources:
+  - centrals
+  - securedclusters
+  verbs:
+  - get
+  - list
+  - watch
+  - patch
+# ACS team can get pod logs
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - pods/log
+  verbs:
+  - get
+  - watch
+  - list
+# ACS team can delete pods
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - delete
+# ACS team can run commands in pods
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - create
```

Comment:
```suggestion
```

Create pod is not a requirement in https://github.com/openshift/managed-cluster-config/pull/2085